### DISCUSSION TITLE
usage on .mov & merging of different file formats

### DISCUSSION THREAD

**Discussion by abe-101** [TGotwig/vidmerger#29] [Q&A] [answered]
1. Is it possible to use this wraper on file with the .mov extenstion?
2. Also is it possible to to combine many file some are .mov and some are .mp4?

**Comment by tgotwig** [comment, marked as answer]
Hey @abe-101 👋
1. I tried it out with [this sample](https://file-examples-com.github.io/uploads/2018/04/file_example_MOV_480_700kB.mov) and it worked 🚀
[README.md](https://github.com/TGotwig/vidmerger/blob/master/README.md) should already tell:

```
    -f, --format <format>    Specifies which formats should be merged individually, the default is
                             👉 avchd,avi,flv,mkv,mov,mp4,webm,wmv
```

2. Nope that doesn't work, but perhaps a good feature for the future 🌈 
Could convert it to the same format into `/tmp` or and then do the merge from there 😺

**Reply by abe-101** [reply]
I like your suggestions, will g-d willing give it a try
